app/reducers/meetings.js

```diff
@@ -54,6 +60,15 @@ export const selectMeetingById = createSelector(
   (meetingsById, meetingId) => meetingsById[meetingId]
 );
 
+export const selectCommentsForMeeting = createSelector(
+  selectMeetingById,
+  state => state.comments.byId,
+  (meeting, commentsById) => {
+    if (!meeting) return [];
+    return (meeting.comments || []).map(commentId => commentsById[commentId]);
```

Comment:
do you need the `|| []` here? if you do it seems like this should be fixed at some other location so it defaults to an empty array

Reply:
Removed it without getting any errors.